workbench/src/Components/LandingPage/LoadDesign/LoadDesign.scss

```
@@ -0,0 +1,203 @@
+.landing-page {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    background: #1a1a1a;
+    overflow-x:auto;
+    font-family: "Inter", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)